config/email.yaml.example

```diff
@@ -1,9 +1,5 @@
 # Outgoing email settings
 
 production:
-  delivery_method: :smtp
-  smtp_settings:
-    address: smtp.example.com
-    port: 25
-    domain: example.com
-    authentication: :none
+  email_delivery:
```

Comment:
Oh sorry, updated.  I didn't change the environment to development when I tested myself :disappointed: